src/common/components/ui/formik/PullSecretField.tsx

```diff
@@ -48,18 +48,15 @@ const PullSecretInfoLink = () => {
   const { t } = useTranslation();
   return (
     <a href={PULL_SECRET_INFO_LINK} target="_blank" rel="noopener noreferrer">
-      {t('ai:Learn more about pull secrets')} <ExternalLinkAltIcon />.
+      {t('ai:Learn more about pull-secrets and view examples')} <ExternalLinkAltIcon />.
```

Comment:
Since this text was already translated to other languages, it there something else we need to do?
How would translators know that they need to update the translations file?

Reply:
@jtomasek Can we change a text that is already translated?

Reply:
Based on discussions, we should perform any needed update to English files and the translation team will sync them.